lib/rspec/core/output_wrapper.rb

```diff
@@ -0,0 +1,11 @@
+require 'delegate'
+
+class OutputWrapper < SimpleDelegator
```

Comment:
Please don't use SimpleDelgegator, it's easy to reimplement and we believe in using as little of the standard library as possible to avoid poisoning a users environment. SimpleDelgator also has performance issues on certain versions of Ruby.

Instead implement the IO public protocol methods and directly delegate to output.